### PR TITLE
planRequests: docs followup, update CHANGELOG.md

### PR DESCRIPTION
**Pull request by Dieterbe**

### CODE REVIEW

File: CHANGELOG.md
Comment (Dieterbe) [reply CHANGELOG.md]: seems worse to imply the change was introduced later. so if someone thinks to upgrade the version "right before this change was introduced" they'll be in trouble. i would keep as is but add a note that you should not use this version but v0.13.1-252-ge2fa083 or higher, instead

Comment (robert-milan) [reply CHANGELOG.md]: Yeah good point, I just don't want them to run 250

Comment (Dieterbe) [reply CHANGELOG.md]: fixed

**Comment by Dieterbe** [comment]
@robert-milan PTAL

File: CHANGELOG.md
Comment (robert-milan) [inline CHANGELOG.md]:
```suggestion
  Note that you should not upgrade to to v0.13.1-250-g21d1dcd1, but to v0.13.1-266-g80ca54f or later due to critical bug fixes.
```
There were 1-2 more bug fixes related to the align requests changes. The shadowed var fix was the latest one and I haven't found any more yet.

Comment (Dieterbe) [reply CHANGELOG.md]: so what do you suggest. we can't possibly always mention all future bugfixes relating to the change we're describing, nor should we. I already added this line to try to accommodate you but frankly we should probably just switch to how it was. One should always operate on the assumption that bugfixes may come after a feature is introduced. If people want as-bugfree-as-possible code they should wait until we tag a stable release.

Comment (robert-milan) [reply CHANGELOG.md]: I agree, but since the fixes are in before this was merged we should probably update it to reflect the latest known version with the bug fixes, or as you suggested just remove it.

Comment (Dieterbe) [reply CHANGELOG.md]: no, a changelog can never accurately describe the current state of affairs, and trying to make it that for this particular case sets the wrong expectations. i just removed this again.

**Review by robert-milan — APPROVED** [review]

**Pull request merged**
